src/mdbExtensionController.ts

```diff
@@ -815,6 +816,44 @@ export default class MDBExtensionController implements vscode.Disposable {
     }
   }
 
+  async showSurveyForEstablishedUsers(): Promise<void> {
+    const surveyId = '9viN9wcbsC3zvHyg7';
+
+    const hasBeenShownSurveyAlready =
+      this._storageController.get(StorageVariables.GLOBAL_SURVEY_SHOWN) ===
+      surveyId;
+
+    // Show the overview page when it hasn't been show to the
+    // user yet, and they have saved connections
+    // -> they haven't just started using this extension
+    if (
```

Comment:
How is the survey connected to the overview page here? This method doesn't trigger showing the overview page, right?

Reply:
Ah, that must be a copy paste leftover 🙈 . I will correct it in the follow up, thanks for noticing!